### PR TITLE
Fix #5814

### LINKED ISSUE #5814: Doc string wrongly says dropout is used.
https://github.com/tensorflow/models/blob/23376e62928486712f6a5cff260c93d0b795bdc0/samples/core/get_started/custom_estimator.py#L30

This line mentions a dropout is used. Still, the code doesn't apply dropout anywhere.  is maybe `learning_rate`?

top-level directory: N/A
OS Platform and Distribution: N/A
TensorFlow installed from: N/A
TensorFlow version: N/A
Bazel version: N/A
CUDA/cuDNN version: N/A
GPU model and memory: N/A
Exact command to reproduce: N/A

### PR DESCRIPTION
**Pull request by johnarevalo**
Comments wrongly say dropout is used

### CODE REVIEW

**Comment by johnarevalo** [comment]
I signed it!

**Comment by lamberta** [comment]
Thank you!
